ccdproc/tests/test_image_collection.py

```diff
@@ -148,6 +148,19 @@ def test_hdus_masking(self, triage_setup):
         new_data = np.array(collection.summary_info)
         assert (new_data == old_data).all()
 
+    def test_multiple_extensions(self, triage_setup):
+        ext1 = fits.PrimaryHDU()
+        ext2 = fits.ImageHDU(name = 'MASK')
+        hdulist = fits.hdu.hdulist.HDUList([ext1, ext2])
+
+        hdulist.writeto(os.path.join(triage_setup.test_dir, 'multi-extension.fits'))
+        ic = image_collection.ImageFileCollection(triage_setup.test_dir, keywords='*',
+                                    filenames = ['multi-extension.fits'], extension='MASK')
```

Comment:
Again no spaces around `=` if it's a keyword parameter.

The indentation is also slightly off. The indentation should match the indentation of the previous line or you could use hanging indentation:

```
ic = ...(
    triage_setup.test_dir, 
    keywords='*', 
    filenames=['multi-extension.fits'], 
    extension='MASK')
```